File: src/ProjectR.Scripting/content/scripts/spells/OpticalCamouflageX69.lua
function GetTargetType()
    return Myself
end

function GetName()
return "Optical Camouflage X69"
end

function GetDescription()
    return "Vanish before your foes.\nIncreases defensive stats, lolumad?"
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .64
end

function GetDelay()
  return .36
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  a:BuffStat(EVA, .66)
  a:BuffStat(DEF, .5)
  a:BuffStat(MR, .5)
  a:BuffStat(SPD, .5)
end

File: src/ProjectR.Scripting/content/scripts/spells/OpticalCamouflageX69.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting.Spells
{
    public class OpticalCamouflageX69 : SpellScriptBase
    {
        public override string Name { get { return "Optical Camouflage X69"; } }
        public override string Description { get { return "Vanish before your foes.\nIncreases defensive stats, lolumad?"; } }

        public override TargetType TargetType { get { return TargetType.Myself; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] {  }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 64; } }
        public override double Delay { get { return 36; } }

        public override void SpellEffect(ICharacter caster)
        {
            BuffStat(Stat.EVA, .66);
            BuffStat(Stat.DEF, .5);
            BuffStat(Stat.MR, .5);
            BuffStat(Stat.SPD, .5);
        }
    }
}